pkg/crcontrollers/drive/drivecontroller_test.go

```diff
@@ -833,11 +833,12 @@ func TestDriveController_handleDriveUsageRemovingWithLED(t *testing.T) {
 	}
 
 	kubeClient := setup()
-	dc := NewController(kubeClient, nodeID, &mocks.MockDriveMgrClient{}, eventRecorder, testLogger)
-	assert.NotNil(t, dc)
-	assert.NotNil(t, dc.crHelper)
 
-	t.Run("Check locate drive state", func(t *testing.T) {
+	t.Run("Check locate drive state - Success", func(t *testing.T) {
```

Comment:
could it be refactored into table tests? I see the code duplication and possibility to use `expectedUsage` and `expectedLEDState` in `tests := []struct{}` (just a proposal, everything else looks good to me)

Reply:
Done